docs/install-upgrade/config.md

```diff
@@ -161,6 +161,7 @@ spec:
               labels: # labels to be added to all logs
                   env: env-1
               url: https://logs-prod-021.grafana.net/loki/api/v1/push
+              useControlProxy: true
```

Comment:
Maybe adding some comment explaining what is it doing would be nice?

Reply:
I added some prose in the preceding section.